docs/src/main/paradox/project/scala3.md

```diff
@@ -1,15 +1,9 @@
 # Scala 3 support
 
-Pekko has experimental support for Scala 3.
+Pekko has support for Scala 3. Now that the Scala 3.3 LTS release is out, we can be pretty confident that it is unlikely that we will need to make binary incompatible changes due to Scala 3 changes. See the @ref:[binary compatibility documentation](../common/binary-compatibility-rules.md).
 
 ## Using 2.13 artifacts in Scala 3
 
-You can use [CrossVersion.for3Use2_13](https://scala-lang.org/blog/2021/04/08/scala-3-in-sbt.html#using-scala-213-libraries-in-scala-3)
+This should not be necessary but you can use [CrossVersion.for3Use2_13](https://scala-lang.org/blog/2021/04/08/scala-3-in-sbt.html#using-scala-213-libraries-in-scala-3)
```

Comment:
We should remove this paragraph entirely, you should only be using the Scala 3 artifacts and considering we are using LTS there shouldn't be an issue here. Also its ideal to not have multiple ways of supporting Scala 3, just use the Scala 3 artifacts.

Also `CrossVersion.for3Use2_13` cannot be used if you are a library that depends on Pekko, it only works with applications.